editoast/src/views/temporary_speed_limits.rs

```diff
@@ -0,0 +1,216 @@
+use axum::extract::Json;
+use axum::extract::State;
+use axum::Extension;
+use chrono::NaiveDateTime;
+use derivative::Derivative;
+use editoast_schemas::infra::DirectionalTrackRange;
+use serde::de::Error as SerdeError;
+use serde::{Deserialize, Serialize};
+use std::result::Result as StdResult;
+use utoipa::ToSchema;
+
+use crate::error::Result;
+use crate::models::temporary_speed_limits::TemporarySpeedLimit;
+use crate::models::Changeset;
+use crate::views::AuthorizationError;
+use crate::views::AuthorizerExt;
+use crate::CreateBatch;
+use crate::Model;
+use editoast_authz::BuiltinRole;
+
+use crate::AppState;
+
+crate::routes! {
+    "/temporary_speed_limits" => create,
+}
+
+editoast_common::schemas! {
+    TemporarySpeedLimitCreateForm,
+    TemporarySpeedLimitCreateResponse,
+    TemporarySpeedLimitItemForm,
+}
+
+#[derive(Serialize, Derivative, ToSchema)]
+struct TemporarySpeedLimitItemForm {
+    start_date_time: NaiveDateTime,
+    end_date_time: NaiveDateTime,
+    track_ranges: Vec<DirectionalTrackRange>,
+    speed_limit: i64,
+    obj_id: String,
+}
+
+#[derive(Serialize, Deserialize, ToSchema)]
+struct TemporarySpeedLimitCreateForm {
+    speed_limits: Vec<TemporarySpeedLimitItemForm>,
+}
+
+#[derive(Serialize, Deserialize, ToSchema)]
+struct TemporarySpeedLimitCreateResponse {
+    pub created_speed_limits_ids: Vec<i64>,
```

Comment:
Prefixing with `created_` is a pattern we don't have in any of our API, I would drop it. In fact, I would probably just put `temporary_speed_limits_ids` or even just `ids` (since we know which endpoint is called anyway). I don't mind either of the two latter solutions (I believe we have kind of both in our API anyway).

Reply:
Thank you for your suggestions. Following a change request from @flomonster, the endpoint will return the id of the newly created `TemporarySpeedLimitGroup` instead: I am updating this conversation to resolved. Otherwise I would have followed your last suggestion and renamed the identifiers vector to `ids` since the nature of the identifiers is indeed implied by the context.